#! /usr/bin/env python
PACKAGE = "image2pcl"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add('image', str_t, 0, 'image file name','')
gen.add('image_res', str_t, 0, 'image res', '1.0')
gen.add('image_x', str_t, 0, 'image origin x', '0.0')
gen.add('image_y', str_t, 0, 'image origin y', '0.0')
gen.add('image_z', str_t, 0, 'image origin z', '0.0')

gen.add('image_roll', str_t, 0, 'image angle roll', '0.0')
gen.add('image_pitch', str_t, 0, 'image angle pitch', '0.0')
gen.add('image_yaw', str_t, 0, 'image angle yaw', '0.0')

gen.add('synchronize', bool_t, 0, 'synchronize image with parameter change', False)
# gen.add('publish', bool_t, 0, 'update image manually', False)

exit(gen.generate(PACKAGE, 'image2pcl', 'image2pcl'))
